Plugins/Object/Object.cpp

```diff
@@ -526,13 +526,22 @@ NWNX_EXPORT ArgumentStack Export(ArgumentStack&& args)
         const auto fileName = args.extract<std::string>();
           ASSERT_OR_THROW(!fileName.empty());
           ASSERT_OR_THROW(fileName.size() <= 16);
+        auto alias = args.extract<std::string>();
```

Comment:
Needs to be the following so it doesn't break prior compiled scripts:

```c++
std::string alias;
try
{
    alias = args.extract<std::string>();
}
catch(const std::runtime_error& e)
{
    LOG_WARNING("NWNX_Object_Export() called from NWScript without sAlias parameter. Please update nwnx_object.nss");
    alias = "NWNX";
}
```

Reply:
Thanks